docs/content/v0.20/example-wecs.md

```diff
@@ -4,29 +4,23 @@ The following steps show how to create new clusters and
 register them with the hub as descibed in the
 [official open cluster management docs](https://open-cluster-management.io/getting-started/installation/start-the-control-plane/).
 
-1. Run the following set of commands for creating a new kind cluster with name "cluster1" and registering it with the
+1. Run the following set of commands for creating two kind clusters with name `cluster1` and `cluster2` and registering it with the
 OCM hub. This cluster will act as a workload cluster.
 
    ```shell
-   export CLUSTER=cluster1
-   kind create cluster --name ${CLUSTER}
-   kubectl config rename-context kind-${CLUSTER} ${CLUSTER}
-   clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${CLUSTER}/" | awk '{print $0 " --context '${CLUSTER}' --force-internal-endpoint-lookup"}' | sh
+   flags="--force-internal-endpoint-lookup" # set this to "" if you have installed KubeStellar on an OpenShift cluster
+   clusters=(cluster1 cluster2);
+   for cluster in "${clusters[@]}"; do
+      kind create cluster --name ${cluster}
+      kubectl config rename-context kind-${cluster} ${cluster}
+      clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${cluster}/" | awk '{print $0 " --context '${cluster}' '${flags}'"}' | sh
+   done   
    ```
 
-   The last line grabs a token from the hub (`imbs1` context), and constructs the command to apply on the new cluster
+   The last line grabs a token from the hub (`imbs1` context), and constructs the command to apply on each cluster
```

Comment:
The "each" part is coming from the loop, not the last line. I think the original text "the new cluster" was good here.

Reply:
ok, updated.